game/scripts/vscripts/modifiers/modifier_boss_capture_point.lua

```diff
@@ -99,12 +99,15 @@ function modifier_boss_capture_point:OnIntervalThink()
     false
   )
   local captureTick
+  local heroMultiplier = 0
 
   -- Start capturing from neutral
   if radiantUnits[1] and self.capturingTeam == nil then
     self.capturingTeam = DOTA_TEAM_GOODGUYS
+    heroMultiplier = #radiantUnits
   elseif direUnits[1] and self.capturingTeam == nil then
     self.capturingTeam = DOTA_TEAM_BADGUYS
+    heroMultiplier = #direUnits
```

Comment:
Also just realised this is actually the wrong place to count the heroes as this block only runs when capture starts from neutral.